sessionctx/variable/session.go

```diff
@@ -2526,6 +2526,8 @@ const (
 	SlowLogIsExplicitTxn = "IsExplicitTxn"
 	// SlowLogIsWriteCacheTable is used to indicate whether writing to the cache table need to wait for the read lock to expire.
 	SlowLogIsWriteCacheTable = "IsWriteCacheTable"
+	// SlowLogStatsLoadStatus is used to indicate the stats load status in memory
+	SlowLogStatsLoadStatus = "StatsLoadStatus"
```

Comment:
According to the design, we won't add a new field. We just extend the content of the current `SlowLogStatsInfoStr` field.

Reply:
updated.